lib/pacemaker/pcmk_output.c

```diff
@@ -175,12 +175,9 @@ rscs_colocated_with_list(pcmk__output_t *out, va_list args) {
         char *hdr = NULL;
 
         if (pcmk_is_set(cons->rsc_lh->flags, pe_rsc_allocating)) {
-            continue;
-        }
-
-        if (recursive) {
             PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "Resources colocated with %s", rsc->id);
-            out->message(out, "rscs-colocated-with-list", cons->rsc_lh, recursive);
+            out->list_item(out, NULL, "%s (id=%s - loop)", cons->rsc_lh->id, cons->id);
+            continue;
```

Comment:
Good catch. BTW we could move the list header before the is-set check since it's identical below now

Reply:
Fixed.